ctx.go

```diff
@@ -363,13 +362,17 @@ func (c *Ctx) Cookie(cookie *Cookie) {
 	fcookie.SetSecure(cookie.Secure)
 	fcookie.SetHTTPOnly(cookie.HTTPOnly)
 
-	switch utils.ToLower(cookie.SameSite) {
-	case "strict":
-		fcookie.SetSameSite(fasthttp.CookieSameSiteStrictMode)
-	case "none":
-		fcookie.SetSameSite(fasthttp.CookieSameSiteNoneMode)
-	default:
-		fcookie.SetSameSite(fasthttp.CookieSameSiteLaxMode)
+	if cookie.SameSite == "" {
```

Comment:
think that this change will cause a breaking change, since the disabled and the lax share the same case

Please add more identifiers for sameSite

like for example "lax" or "disabled"

but please make sure that no breaking change occurs and the identifiers are documented to the cookie struct

furthermore additional unittests for the new cases would be good

Reply:
please check code again~